Makefile

```diff
@@ -127,6 +127,8 @@ godoc: install
 # --- frontend -----------------------------------------------------------------
 NODEPREFIX = .hermit/node
 NODELIB = $(NODEPREFIX)/lib
+# TODO: BASEURL should be calculated dynamically for CI/PR.
+export BASEURL ?= https://evy.dev
```

Comment:
Is it your intention that if BASEURL is set in the environment when `make` is invoked that the environment variable override this one? If not, there is no need for `?=`. You can use just the normal `=`.

Reply:
yup, that was on purpose, I've been running it with:

    BASEURL=http://localhost:8080 make e2e

for local testing of the make target. Either way, I suspect it is temporary.

Reply:
The risk here is that someone has `BASEURL` set in their environment for other reasons and it ends up overriding the value here. You should try to get used to setting make variables on the command line and not environment variables - i.e. `make e2e BASEURL=http://localhost:8080`. If you do that, you do not set the var in the environment before make, and make will override a var in the makefile of the form `BASEURL = ...` (i.e. a standard make assignment).

Reply:
yeah, that makes sense, i've changed it to `=`